dev/io.openliberty.jakarta.jsonp.2.1_fat_tck/publish/tckRunner/tck/pom.xml

```diff
@@ -81,6 +81,7 @@
 						<version>3.1.2</version>
```

Comment:
Should this version also be updated to 3.5.2

Reply:
Yes

Reply:
Good spot, thanks. I've cancelled and re-requested builds.